internal/obsreportconfig/obsreportconfig.go

```diff
@@ -31,6 +31,14 @@ var UseOtelForInternalMetricsfeatureGate = featuregate.GlobalRegistry().MustRegi
 	featuregate.StageAlpha,
 	featuregate.WithRegisterDescription("controls whether the collector uses OpenTelemetry for internal metrics"))
 
+// EnableHighCardinalityMetricsfeatureGate is the feature gate that controls whether the collecor should enable
+// potentially high cardinality metrics.
+var EnableHighCardinalityMetricsfeatureGate = featuregate.GlobalRegistry().MustRegister(
+	"telemetry.enableHighCardinalityMetrics",
```

Comment:
What about renaming this as "enableRPCMetrics" or something like that, to be more specific about what this is gating?

Reply:
this was coming from the issue i opened against the collector https://github.com/open-telemetry/opentelemetry-collector/issues/7517#issuecomment-1511357467 given there may be some more high cardinality metrics like this in the future.